exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (true) {
```

Comment:
This doesn't make sense. True will always evaluate true and so the else statement will never be reached. I would instead do something like:

if(danielsRole === "mentor")